File: 691.pas
program prog691;

var
normsim: string = 'ABCEHKMOPTXY';
normnum: string = '0123456789';
      s: string;
      t: integer;

begin
readln(t);
for var i := 1 to t do
  begin
  readln(s);
	if (Length(s) = 6) and
	  (copy(s, 1, 1)  in normsim) and
	  (copy(s, 5, 1) in normsim) and
	  (copy(s, 6, 1) in normsim) and
	  (copy(s, 2, 1) in normnum) and
	  (copy(s, 3, 1) in normnum) and
	  (copy(s, 4, 1) in normnum) then
		writeln('Yes')
	else
		writeln ('No');
	end;
end.

File: 691.py
normsim = ['A', 'B', 'C', 'E', 'H', 'K', 'M', 'O', 'P', 'T', 'X', 'Y']
normnum = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
for i in range(int(input())):
	s = input()
	if  len(s) == 6 and s[0] in normsim and s[4] in normsim and s[5] in normsim and s[1] in normnum and s[2] in normnum and s[3] in normnum:
		print('Yes')
	else:
		print('No')
